### PR TITLE
feat: Add possibility to disable maintainance warning

### PR DESCRIPTION
**Pull request by aragon999**
If you have noted the message I think one should be able to disable it. At least the `sleep 1`.

Not sure if some tests are needed for this?

### CODE REVIEW

File: aurget [hunk 1]
```diff
@@ -797,11 +798,13 @@ declare -A targets         # pkgbase->name(s)
 declare -A target_versions # name->version
 declare -A target_deps     # pkgbase->is-dep?
 
-warn "aurget is no longer being maintained, if you would like to volunteer please communicate your interest on this GitHub Issue: https://github.com/pbrisbin/aurget/issues/66"
-sleep 1
-
 initialize "$@"
 
+if [ ! "${disable_maintenance_warning}" = "true" ]; then
+  warn "aurget is no longer being maintained, if you would like to volunteer please communicate your interest on this GitHub Issue: https://github.com/pbrisbin/aurget/issues/66"
```
Comment (pbrisbin) [inline aurget:804]:
```suggestion
  warn "aurget is no longer being maintained, if you would like to volunteer please communicate your interest on this GitHub Issue: https://github.com/pbrisbin/aurget/issues/66. You can disable this warning by setting disable_maintenance_warning=true in your aurgetrc."
```

File: aurget [hunk 2]
```diff
@@ -797,11 +798,13 @@ declare -A targets         # pkgbase->name(s)
 declare -A target_versions # name->version
 declare -A target_deps     # pkgbase->is-dep?
 
-warn "aurget is no longer being maintained, if you would like to volunteer please communicate your interest on this GitHub Issue: https://github.com/pbrisbin/aurget/issues/66"
-sleep 1
-
 initialize "$@"
 
+if [ ! "${disable_maintenance_warning}" = "true" ]; then
```
Comment (pbrisbin) [inline aurget:803]:
```suggestion
if [[ ! "$disable_maintenance_warning" == "true" ]]; then
```

**Review by pbrisbin — APPROVED** [review]
No worries on the tests, IMO.

**Comment by aragon999** [comment]
Perfect. Updated the pull request according to your suggestions.

**Comment by pbrisbin** [comment]
Released in v4.7.5.

**Pull request merged**